create-k8s-chained-sessions.sh

```diff
@@ -90,7 +96,8 @@ PARENT_SESSION_NAMES="panorama-k8s-playground panorama-k8s-playground-2 panorama
 
 for session in $PARENT_SESSION_NAMES
 do
-    createLeappSession "$session" "AWSAdministratorAccess" "eks-admin-1.24"
-    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama-dev-writer-1.24"
-    createLeappSession "$session" "PanoramaK8sEngineeringDefault" "panorama-dev-reader-1.24"
+    createLeappSession "$session" "AWSAdministratorAccess" "eks" "admin" "1.24"
```

Comment:
Trying to reign in scope creep for now, so I left the `eks` and `panorama` keywords, but only to support the existing IAM roles.